aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/ConnectEvent.java

```diff
@@ -72,6 +72,33 @@ public static class CustomerEndpoint implements Serializable, Cloneable {
         private String type;
     }
 
+        @Data
+    @Builder(setterPrefix = "with")
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class MediaStreams implements Serializable {
```

Comment:
Looking at the event you pasted in the PR's description, you should add `MediaStreams` class to the `ContactData` class